src/AbstractPathMappingRepository.php

```diff
@@ -278,4 +300,49 @@ protected function createVirtualResource($path = null)
 
         return $resource;
     }
+
+    /**
+     * Transform a relative path into an absolute path.
+     *
+     * @param string $absolutePath
+     *
+     * @return string
+     */
+    protected function makePathRelative($absolutePath)
+    {
+        return Path::makeRelative($absolutePath, $this->baseDirectory);
```

Comment:
Let's remove this method and use `Path::makeRelative()` directly. Less method calls = better (in PHP).